src/hooks/useActions.ts

```diff
@@ -738,11 +739,17 @@ export default function useActions() {
   )
 }
 
+type AppFunctions = 'executeAdjustment' | 'executeVote' | 'resolve'
```

Comment:
i wouldn't define which app functions for this one because is something really generic, at some point we might use it a lot to do different contract calls and we don't want to add a bunch of funcion names to that type

Reply:
Will change it to string. Thats ok?

Reply:
yeah